x-pack/plugins/cloud_security_posture/public/common/navigation/use_csp_breadcrumbs.ts

```diff
@@ -27,47 +23,53 @@ const getClickableBreadcrumb = (
   }
 };
 
-export const useCspBreadcrumbs = (breadcrumbs: CspNavigationItem[]) => {
+export const useCspBreadcrumbs = (breadcrumbs: BreadcrumbEntry[]) => {
   const {
     services: {
       chrome: { setBreadcrumbs, docTitle },
-      application: { getUrlForApp },
+      application: { currentAppId$, applications$, navigateToApp },
     },
   } = useKibana<CoreStart>();
+
   const match = useRouteMatch();
-  const history = useHistory();
+
+  const appId = useObservable(currentAppId$);
+  const applications = useObservable(applications$);
+  const application = appId ? applications?.get(appId) : undefined;
+  const appTitle = application?.title;
```

Comment:
NIT: If this breadcrumbs are only used for the CSP plugin component, we could use `PLUGIN_NAME` constant directly, it is the same value passed to the application title.

Reply:
They are also used in the security solution but only for the "Rules" page, so it needs to work for both cases